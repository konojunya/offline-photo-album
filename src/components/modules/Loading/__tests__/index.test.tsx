import * as React from "react";
import * as renderer from "react-test-renderer";
import { Loading } from "..";
describe("Loading component testing", () => {
  test("Loading component snapshot", () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
