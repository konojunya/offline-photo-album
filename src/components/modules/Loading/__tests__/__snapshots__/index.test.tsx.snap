// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component testing Loading component snapshot 1`] = `
<p>
  loading...
</p>
`;
